heartbeat/docker-compose

```diff
@@ -185,19 +196,25 @@ docker_compose_status()
 
 docker_compose_start()
 {
+
+	sleep 5
```

Comment:
You shouldnt need to sleep here.

Reply:
yes, agree. I am still working on it... it could be improved in the future version.